src/main/java/uk/gov/hmcts/reform/professionalapi/controller/response/ProfessionalUsersEntityResponseWithoutRoles.java

```diff
@@ -11,6 +12,9 @@
 @NoArgsConstructor
 public class ProfessionalUsersEntityResponseWithoutRoles {
 
+    @JsonProperty
+    private String organisationIdentifier;
```

Comment:
do we required organisationIdentifier in both classes? (super and sub classes)

Reply:
There is no parent-child relationship here

Reply:
ok. i saw some code before thats why asked..